app/(root)/page.tsx

```
@@ -1,7 +1,49 @@
+          src="/robot.png"
+          alt="Robo-dude"
+          width={400}
+          height={400}
+          className="max-sm:hidden"
+        />
+      </section>
+      <section className="flex flex-col gap-6 mt-8">
+        <h2>Your Interviews</h2>
+        <div className="interviews-section">
+          {dummyInterviews.map((interview) => (
+            <InterviewCard {...interview} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)